src/planwise/client/projects2/components/settings.cljs

```diff
@@ -178,19 +193,13 @@
          [:section {:class-name "project-settings-section"}
           [section-header 5 "Actions"]
           [:div {:class "project-setting"}
-           [:div
-            [:p [m/Icon "account_balance"] "Available budget"]]
-           [current-project-input "" [:config :actions :budget] "number" {:disabled read-only :class "project-setting"}]
-           [m/TextFieldHelperText {:persistent true} "Planwise will keep explored scenarios below this maximum budget"]
-
-           [:div [:p [m/Icon "domain"] "Building a new provider..."]]
-           [listing-actions :build @build]
-
-           [:div [:p [m/Icon "arrow_upward"] "Upgrading a provider so that it can satisfy demand would cost..."]]
-           [current-project-input "" [:config :actions :upgrade-budget] "number" {:disabled read-only :class "project-setting"}]
-
-           [:div [:p [m/Icon "add"] "Increase the capactiy of a hospital by..."]]
-           [listing-actions :upgrade @upgrade]]]]]])))
+           [action-option {:type :budget}]
+           [action-option {:type :build
+                           :disabled? read-only
+                           :list @build-actions}]
+           [action-option {:type :budget-upgrade}]
+           [action-option {:type :upgrade
+                           :list @upgrade-actions}]]]]]])))
```

Comment:
What is the purpose of the `action-option` function? It seems to me it obfuscates the code. I find the expanded UI simpler and more readable.